Assets/SEE/UI/Window/MetricWindow.cs

```diff
@@ -0,0 +1,188 @@
+                }
+            }
+            else
+            {
+                var searchList = Search(str, elements);
+                foreach (GameObject ele in elements)
+                {
+                    if (ele != null)
+                    {
+                        string eleText = ele.transform.GetChild(0).GetComponent<TextMeshProUGUI>().text;
+
+                        if (searchList.Contains(eleText))
+                        {
+                            ele.SetActive(true);
+                        }
+                        else
+                        {
+                            ele.SetActive(false);
+                        }
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Performs a fuzzy search
+        /// </summary>
+        /// <param name="query"> the search query </param>
+        /// <param name="ObjectList"> array of GameObjects containing the objects to search through</param>
+        /// <param name="cutoff"> representing the cutoff score for relevance </param>
+        /// <returns> An iterable collection of strings representing the attributes of the GameObject instances, ordered by relevance to the search query </returns>
+        public IEnumerable<string> Search(string query, GameObject[] ObjectList, int cutoff = 62)
+        {
+            string[] attributesList = new string[ObjectList.Length];
+
+            for(int i = 0; i < ObjectList.Length; i++)
+            {
+                attributesList[i] = ObjectList[i].transform.GetChild(0).GetComponent<TextMeshProUGUI>().text;
+            }
+
+            var listTest = Process.ExtractAll(query, attributesList, cutoff: cutoff);
+            IEnumerable<(int score, string attribute)> listTest2 = listTest.Select(x => (x.Score, x.Value));
+
+            listTest2 = listTest2.OrderByDescending(x => x.score);
+
+            return listTest2.Select(x => x.attribute);
+        }
+
+        /// <summary>
+        /// Creates the Metric Window
+        /// </summary>
+        /// <param name="graphElement"></param>
+        public void CreateUIInstance()
+        {
+            //Instantiate MetricWindow
+            MetricWindowObject = PrefabInstantiator.InstantiatePrefab(SettingsPrefab, Window.transform.Find("Content"), false);
+            MetricWindowObject.name = "Scrollable";
+
+            //Parent Content
+            Transform ScrollViewContent = MetricWindowObject.transform.Find("Content/Items").transform;
+
+            //Input Field
+            TMP_InputField inputField = MetricWindowObject.transform.Find("Search/SearchField").gameObject.MustGetComponent<TMP_InputField>();
+
+            inputField.onSelect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = false);
+            inputField.onDeselect.AddListener(str => SEEInput.KeyboardShortcutsEnabled = true);
+
+            //Int Attributes
+            foreach (KeyValuePair<string, int> kvp in graphElement.IntAttributes)
```

Comment:
I am pretty sure this `foreach` loop and the next one can be turned into a non-redundant method. Avoid copying and pasting code.

Reply:
I created a new method "DisplayAttributes" to avoid the redundant code